piazza_api/rpc.py

```diff
@@ -127,6 +127,18 @@ def content_instructor_answer(self, params):
         return self._handle_error(r, "Could not create object {}.".format(
                                      repr(params)))
 
+    def content_mark_duplicate(self, params):
+        """Mark a post as duplicate to another.
+
+        :type params: dict
+        :param params: the parameters to be passed in
+        """
+        r = self.request(
+            method="content.duplicate",
+            data=params
+        )
+        return self._handle_error(r, "Could not create object {}.".format(
+                                     repr(params)))
```

Comment:
I don't know whether u get my previous comment or not but I used the convention of `content_instructor_answer` and `content_create`. I think the style should be consistent through out the file